### ISSUE TITLE
Training log

### ISSUE THREAD

**Issue by flyyyyer** [JinluZhang1126/MixSTE#15]
Could you provide the traning log and the pretrained model? It seems hard to convergence.

**Comment by JinluZhang1126** [comment]
Can I ask you how many epochs you have trained?

**Comment by JinluZhang1126** [comment]
> Could you provide the traning log and the pretrained model? It seems hard to convergence.

I can not attach the code anytime now, but the training log and checkpoint will be supplied as soon.

There are some info that may help you: I trained for about 30 hours in 2080Ti GPU to get convergence, and the training loss went down after about 2-4 epochs.

**Comment by JinluZhang1126** [comment]
> Can I ask you how many epochs you have trained?

I can not attach the code anytime now, but the training log and checkpoint will be supplied as soon.

There are some info that may help you: I trained for about 30 hours in 2080Ti GPU to get convergence, and the training loss went down after about 2-4 epochs.
